internal/core/src/storage/DiskFileManagerImpl.cpp

```diff
@@ -58,22 +58,22 @@ using WriteLock = std::lock_guard<std::shared_mutex>;
 
 namespace milvus::storage {
 
-DiskANNFileManagerImpl::DiskANNFileManagerImpl(const FieldDataMeta& field_mata, const IndexMeta& index_meta)
+DiskFileManagerImpl::DiskFileManagerImpl(const FieldDataMeta& field_mata, const IndexMeta& index_meta)
```

Comment:
One question: will we create S3 client everytime we build diskann index or query on diskann index?

Reply:
> One question: will we create S3 client everytime we build diskann index or query on diskann index?

MinioChunkmanger and localchunkManager is single instance now, so not.

Reply:
we will set s3 client to single instance later, because index tasks have diffrent storage config in indexpool, so every index will has a independent minioChunkManager.